### PR TITLE
Add Cliff examples

### PR DESCRIPTION
**Pull request by scottcandy34**
- Add examples for Root and Create 3

### CODE REVIEW

File: examples/root_robots/cliff_sensor.py
Comment (scottcandy34) [inline examples/root_robots/cliff_sensor.py:8]: Oops this needs to change from the Create3 to Root

Comment (scottcandy34) [reply examples/root_robots/cliff_sensor.py]: Fixed

File: examples/create3_robots/cliff_sensors.py
Comment (shamlian) [inline examples/create3_robots/cliff_sensors.py:2]: (c) 2024

File: examples/root_robots/cliff_sensor.py
Comment (shamlian) [inline examples/root_robots/cliff_sensor.py:2]: (c) 2024

File: examples/create3_robots/cliff_sensors.py
Comment (shamlian) [inline examples/create3_robots/cliff_sensors.py:6]: No need to include `Color` or `Root`

File: examples/root_robots/cliff_sensor.py
Comment (shamlian) [inline examples/root_robots/cliff_sensor.py:6]: no need to include `Color`

**Review by shamlian — CHANGES_REQUESTED** [review]
Some small changes requested, otherwise good to go. Thank you!

File: examples/create3_robots/cliff_sensors.py
Comment (scottcandy34) [reply examples/create3_robots/cliff_sensors.py:6]: I Just copied it from the other examples, otherwise I would of removed them. You might want to go check all the other examples for that as well. But I will remove it no problem.

Comment (shamlian) [reply examples/create3_robots/cliff_sensors.py:6]: Agreed -- we are inconsistent -- not a judgement on you. Just trying to fix them as we touch files.

Comment (scottcandy34) [reply examples/create3_robots/cliff_sensors.py:6]: Gotcha. Been there.

File: examples/root_robots/cliff_sensor.py
Comment (scottcandy34) [reply examples/root_robots/cliff_sensor.py:6]: What about `hand_over` and `Robot` both are note used as well.

Comment (shamlian) [reply examples/root_robots/cliff_sensor.py:6]: That's true. I'd say you are welcome to remove those, as well.

**Review by shamlian — APPROVED** [review]
I will merge this after we get #37 merged. Thanks again, and happy Friday!

**Pull request merged**
